examples/t_cose_basic_example_ossl.c

```diff
@@ -449,6 +449,7 @@ int two_step_sign_example(void)
      * message. For that call the payload is just passed in as a
      * buffer.
      */
+    QCBOREncode_BstrWrap(&cbor_encode);
```

Comment:
The examples where rearranged, so this file doesn't exist any more, but you can find the same example in signing_examples.c. The same code is use for both PSA and Mbed TLS so you only need one change.